lib/ueberauth/config/config.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Ueberauth.Config do
+  @moduledoc false
```

Comment:
Add some documentation, especially that this is a public API

Reply:
This whole module seems rather complicated.  Couldn't we do away with the `Module.put_attribute/3` and just do:

```elixir
def json_library do
  Application.get_env(:ueberauth, :json_library)
end
```